templates/withKibana/epochInfo.go

```diff
@@ -0,0 +1,12 @@
+package withKibana
+
+// EpochInfo will hold the configuration for the epochinfo index
+var EpochInfo = Object{
```

Comment:
no `epochInfo_policy` needed?

Reply:
no need because. this index will be small and don't need policy.